CHANGELOG.md

```diff
@@ -55,6 +55,13 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 - Added support for `.gitignore` ignored files and directories when transferring
   repo in `wharf run`. Can be disabled via new `--no-gitignore` flag. (#85)
 
+- Added input variables support, and with `--input, -i` flag to `wharf run` and
+  `wharf vars` commands that allows setting values, ex: (#97)
+
+  ```sh
+  wharf run --input myInputVar=myValue
+  ```
```

Comment:
nit: Would be nice if there was an example with multiple input variables. Probably no need for a single value example in that case.

Is it
`--input myInputVar1=myValue1,myInputVar2=myValue2`
or
`--input myInputVar1=myValue1 --input myInputVar2=myValue2`?

And what happens with duplicate keys?

Better place for usage specifics would probably be in docs or something, but at least having >1 values in the example would be nice.

Reply:
On duplicated values the latter will override the former, which IMO is expected.

Good idea with the multiples examples though!

```suggestion
  ```sh
  wharf run --input myInputVar=myValue
  # => [ "myInputVar": "myValue" ]

  # Supports multiple:
  wharf run --input var1=value1 --input var2=value2
  # => [ "var1": "value1", "var2": "value2" ]

  # On collisions, the last value is used:
  wharf run --input myVar=foo --input myVar=bar
  # => [ "myVar": "bar" ]
  ```
```

Reply:
How about now?